dodge_the_creeps_3d_c#/Main.cs

```diff
@@ -0,0 +1,44 @@
+using Godot;
+using System;
+
+public class Main : Node
+{
+#pragma warning disable 649
+	[Export]
+	public PackedScene MobScene;
+#pragma warning restore 649
+
+	public override void _Ready()
+	{
+		GD.Randomize();
```

Comment:
As per the [C# style guide](https://docs.godotengine.org/en/latest/tutorials/scripting/c_sharp/c_sharp_style_guide.html), use 4 spaces instead of tabs to follow the C# language standard.